migration-tool/src/main/resources/scripts/__init__.py

```diff
@@ -0,0 +1 @@
+
```

Comment:
Is this intentionally blank ?

Reply:
Yes, it's used to mark this directory as a Python package.